modules/Bio/Vega/DBSQL/TranscriptAdaptor.pm

```diff
@@ -276,6 +277,28 @@ sub remove {
 
 1;
 
+sub add_persistent_attributes {
+    my($self, $tran) = @_;
+
+    #Find attributes for the transcript and add them to the transcript
+    my $stable_id = $tran->stable_id;
+    my @persistent_attributes = ('vega_name', 'TAGENE_transcript', 'MANE_Select', 'ccds_transcript', 
```

Comment:
Because this list of attributes is reused in multiple place, it might be worth having it only one place